_source/_posts/2022-08-24-state-of-project-loom.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: blog_post
+title: "What the Heck Is Java Project Loom?"
+author: deepu-sasidharan
+by: advocate
+communities: [java]
+description: "What the heck is Java Project Loom and what does it mean for average Java developers?"
```

Comment:
```suggestion
description: "What the heck is Java's Project Loom and what does it mean for average Java developers?"
```

I'm not sure using "average" is an inclusive term. Maybe "everyday" or "regular" would be better? Or just remove it.